routes/resources.js

```diff
@@ -57,6 +57,16 @@ module.exports = function (app) {
       .catch((error) => handleError(res, error, params))
   })
 
```

Comment:
I know we want to use swagger on this eventually but in the meantime can we just put a comment here about how the a client would format its query params?